extensions/productboard/src/index.tsx

```diff
@@ -52,19 +52,54 @@ export default function Command() {
     }
   }
 
-  return (
+  const { FORM_FIELDS } = preferences;
+
+  if (!FORM_FIELDS) return (
```

Comment:
There is a lot of repeated code with this approach. Would it be possible to clean it up a bit, so e.g. title, tags and saidBy fields are not defined twice?